tc/app/laravel/functions

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env bash
+
+# shellcheck disable=SC1091
+
+source "$CLI_DIR/tc/framework/functions"
+
+function _get_bitbucket_access_token() {
+  __token_file=token-$(date +'%Y-%m-%d_%H-%M-%S').json
+  __bitbucket_client_id="$1"
+  __bitbucket_client_secret="$2"
+  curl -s -X POST \
+      https://bitbucket.org/site/oauth2/access_token \
+      -H 'Cache-Control: no-cache' \
+      -H 'Content-Type: application/x-www-form-urlencoded' \
+      -d "client_id=${__bitbucket_client_id}&grant_type=client_credentials&client_secret=${__bitbucket_client_secret}" \
+      -o "$__token_file"
+  _error_code=$?
+  if [ $_error_code -eq "0" ]; then
+    __access_token=$(jq -r '.access_token' "$__token_file")
+    export __access_token
+    rm "$__token_file"
+  else
+    __MSG="Error getting bitbucket access token"
+    _message "$__MSG" "error" "$_error_code"
+  fi
+}
+
+function _send_pull_request() {
+  __site_version=$(grep "^APP_VERSION" .env | awk -F "=" '{ print $2 }')
+  __bitbucket_username="$1"
+  __repo_slug="$2"
+  __branch_from="$3"
+  __branch_to="$4"
+  __response_file="$5"
```

Comment:
why you send this parameter? This is an inner temporary file that you delete at the end of the function.